src/components/ui/slider/slider.tsx

```diff
@@ -55,24 +57,38 @@ export const Slider = ({
   disabled,
   min = 1,
   max = 3,
+  level = "beginner",
 }: SliderProps) => {
   const steps = max - min
+  let backgroundWidth
+  const isMobile = useMobileView()
+  if (level === "beginner") {
+    backgroundWidth = ((sliderValue ?? min) - min) * (100 / steps)
+  } else if (level === "intermediate") {
+    backgroundWidth = (Math.min(sliderValue ?? min, 2) - min) * (100 / steps)
+  } else {
+    backgroundWidth = 100
+  }
```

Comment:
Can we change this so that we can make it more reusable? Calculate the width using the min and max values instead of passing a `level` prop?